go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/DataDog/zstd v1.5.2
 	github.com/Microsoft/go-winio v0.5.2
 	github.com/NYTimes/gziphandler v1.1.1
-	github.com/ava-labs/coreth v0.12.9-rc.5
+	github.com/ava-labs/coreth v0.12.9-rc.5.0.20231128173117-e4307e460503
```

Comment:
I think we need to rebase coreth off this PR and update the coreth dependency here. But everything here lgtm